src/vma/proto/route_table_mgr.cpp

```diff
@@ -290,6 +291,7 @@ void route_table_mgr::parse_attr(struct rtnl_route *route, route_val *p_val)
 	}
 
 	// Table ID
+	/* coverity[check_return] */
 	int table_id = rtnl_route_get_table(route);
```

Comment:
`rtnl_route_get_table()` is simply and accessor function, what is the warning here?